packages/manager/apps/pci-gateway/src/i18n.ts

```diff
@@ -18,7 +18,10 @@ export default async function initI18n(
     .init({
       lng: locale,
       fallbackLng: 'fr_FR',
-      supportedLngs: availablesLocales,
+      supportedLngs: availableLocales,
+      interpolation: {
+        escapeValue: false,
+      },
```

Comment:
You should not duplicate the i18n configuration in each project
packages/manager/core/shell-client/src/initI18n.ts already contain this configuration and can be imported directly from `@ovh-ux/manager-react-shell-client`

example usage : 
```
  const context = await initShellContext(appName, getTrackingContext(appName));
  await initI18n({
    context,
    reloadOnLocaleChange: true,
    defaultNS: appName,
    ns: [appName, `${appName}/listing`],
  });
```